src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,53 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 10px;
+  gap: $gap-card;
+  width: getWidth($box-size, $cards-per-row, $gap-card);
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+    @include square($box-size);
+    &:hover {
+      background-color: #ffc0cb;
+      position: relative;
+      bottom: 20px;
+      cursor: pointer;
+      transition-duration: 500ms;
```

Comment:
Let's add transition rules not on hover, but on whole card. And also you need to specify to which properties this transition-duration is applied.